Gemfile.lock

```diff
@@ -15,10 +15,25 @@ GEM
     coderay (1.1.3)
     diff-lcs (1.4.4)
     method_source (1.0.0)
+<<<<<<< HEAD
     mini_mime (1.1.0)
+    mini_portile2 (2.5.3)
+    nokogiri (1.11.3)
+      mini_portile2 (~> 2.5.0)
+      racc (~> 1.4)
+    nokogiri (1.11.3-x64-mingw32)
+      racc (~> 1.4)
     nokogiri (1.11.3-x86_64-darwin)
+=======
+    mini_mime (1.1.1)
+    mini_portile2 (2.6.1)
+    nokogiri (1.12.3)
+      mini_portile2 (~> 2.6.1)
+>>>>>>> upstream/master
+      racc (~> 1.4)
```

Comment:
this is not resolved conflict results
======= and >>>>>>> upstream/master shouldn't be here

Reply:
Done